tutorials/gui/gui_containers.rst

```diff
@@ -0,0 +1,181 @@
+.. _doc_gui_containers:
```

Comment:
Added by reduz in dbf5a855d5, from a quick look it doesn't seem to have 3.2-only stuff.